src/components/button/Button.driver.ts

```diff
@@ -17,7 +17,7 @@ export class ButtonDriver extends ComponentDriver {
   }
 
   isPressable = async () => {
-    if (this.exists()) {
+    if (await this.exists()) {
```

Comment:
I think Miki just change the onPress to be sync as it actually is sync. Don't we need to remove the async instead?

Reply:
https://github.com/wix/react-native-ui-lib/pull/2459/files
It was merged after the Adi cut the version (I wasn't aware that we're cutting a version yesterday) so maybe we should include it in v7

Reply:
@Inbal-Tish This fix is relevant. The isExists is async and we address it a sync function..

Reply:
As Inbal stated it looks like it was merged to master after we started working on v7.

Reply:
Correct

Reply:
I don't see any delta. 
Which change are you referring to exactly? why did we merged changes while working on v7? 
anyway, do we need to cherry pick it to v7?

Reply:
I merged the PR since no one told me that we were cutting a version. The change is in the PR I added here